sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -329,8 +279,3 @@ def aggregated_values(self, aggregator_or_name):
 
   def metrics(self):
     return self._evaluation_context.metrics()
-
-
-class EagerRunner(DirectRunner):
```

Comment:
<!--thread_id:cc_164250769_t; commit:0f554fd51414381646c32b36f6f87a14adc0fad8; resolved:0-->
<!--section:context-quote-->
> **robertwb** wrote:
> We still might need this (see, e.g. https://github.com/apache/beam/pull/3906)

<!--section:body-->
I don't quite understand the relationship here--can you explain?  It seems like eagerness should be a property we support for runners, rather than explicitly having a particular eager runner.